src/Chapter1.hs

```diff
@@ -508,7 +522,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = min (abs x) (abs y)
```

Comment:
This is a cool idea! 💡 
However, this solution doesn't preserve the sign of numbers, So it would return the positive number, even when it worked with the negative ones.

Reply:
Resolved.